src/pages/activities-classic/view/activities-classic-view.page.ts

```diff
@@ -0,0 +1,88 @@
+import { Component } from '@angular/core';
+import { ModalController, NavParams, NavController } from 'ionic-angular';
+import { ActivitiesClassicViewModalPage } from './activities-classic-view-modal.page';
+import { AssessmentsPage } from '../../assessments/assessments.page';
+
+@Component({
+  templateUrl: './view.html'
+})
+
+export class ActivitiesClassicViewPage {
+  activity: any = {};
+  submissions: Array<any> = [];
+
+  constructor(
+    private navParams: NavParams,
+    private navCtrl: NavController,
+    private modalCtrl: ModalController
+  ) {
+  }
+
+  // @TODO: use simple mock data for assessment first
+  /**
+   * on assessment implementation, to do list:
+   * - load badges
+   * - change icon display based on responded data format
+   * - load submission into this.submissions
+   * - change template view based on responded data format
+   */
+  ionViewDidEnter(): void {
+    this.activity = this.navParams.get('activity');
+    this.activity.badges = [
+      {
+        url: 'http://leevibe.com/images/category_thumbs/video/19.jpg',
```

Comment:
@PhilippLaufenberg no, we dont. But since the achievement listing is nearly done in the frontend. which it [extracts badges from achievements](https://github.com/intersective/practera-sdk-ionic/blob/feature/ISDK-41/checkin/src/pages/activities/list/list.page.ts#L90-L94), so we can then remove those sample badges images.